app/controllers/presentations_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.page(params[:page]).order(:created_at)
+    @presentations = Presentation.page(params[:page]).order(created_at: :desc)
```

Comment:
We had this conversation yesterday as a pod. I feel strongly that I want to see the newest created presentations at the top. For instance there might be a presentation that is being created for way in the future but I want to get excited about it on creation. I want to see new content.

We have a route to do with events called `events/upcoming` I feel like the same route for presentations would be a good place to show upcoming presentations but this being an index page I want to see the newest available content at the top. Makes me feel like there is a reason to be re-visiting this page.